lua/neotest-plenary/init.lua

```diff
@@ -86,16 +90,24 @@ function PlenaryNeotestAdapter.build_spec(args)
       table.insert(filters, 1, parent_pos.range[1])
     end
   end
-  local min_init
-  for _, path in ipairs(async.fn.glob(("**%sminimal_init*"):format(lib.files.sep), true, true)) do
-    min_init = path
-  end
+  local min_init = config.min_init
   if not min_init then
-    for _, path in ipairs(async.fn.glob(("test*%sinit.vim"):format(lib.files.sep), true, true)) do
-      min_init = path
+    local globs = {
+      ("**%testrc*"):format(lib.files.sep),
+      ("**%sminimal_init*"):format(lib.files.sep),
+      ("test*%sinit.vim"):format(lib.files.sep),
+    }
+    for _, pattern in ipairs(globs) do
+      local glob_matches = async.fn.glob(pattern, true, true)
+      if #glob_matches > 0 then
+        min_init = glob_matches[1]
+        goto min_init_found
```

Comment:
Use `break` rather than `goto`, which suits the need better and `goto` isn't available in all lua versions that neovim can use https://github.com/dstein64/nvim-scrollview/pull/75.

Reply:
I have no idea why I thought Lua didn't have `break` :sweat_smile:

Reply:
ah, lua has `break` but not `continue`